plugin.rb

```diff
@@ -114,7 +115,10 @@
     Mustache.render(template, args).strip
   end
 
-  add_to_serializer(:current_user, :can_create_jira_issue, false) { true }
+  add_to_serializer(:current_user, :can_create_jira_issue, false) do
```

Comment:
You can use the `include_condition` argument here to include only if SiteSetting.discourse_jira_enabled is on.

```suggestion
  add_to_serializer(:current_user, :can_create_jira_issue, include_condition: -> { SiteSetting.discourse_jira_enabled }) do
    user&.in_any_groups?(SiteSetting.discourse_jira_allowed_groups_map)
  end
```